legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-h2/src/main/java/org/finos/legend/engine/persistence/components/relational/h2/sqldom/schemaops/values/Udf.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.persistence.components.relational.h2.sqldom.schemaops.values;
+
+import org.finos.legend.engine.persistence.components.relational.sqldom.schemaops.values.Function;
+import org.finos.legend.engine.persistence.components.relational.sqldom.schemaops.values.Value;
+
+import java.util.ArrayList;
+
+public class Udf extends Function
+{
+    private String udfName;
+
+    public Udf(String quoteIdentifier, String udfName)
```

Comment:
Let's move it to core and share it with Snowflake as well

Reply:
Sure, changed